cli/types/index.d.ts

```diff
@@ -1803,6 +1803,19 @@ declare namespace Cypress {
     whitelist: string | string[] | RegExp | ((cookie: any) => boolean)
   }
 
+  /**
+   * Options that control how a command behaves in the `within` scope

```

Comment:
Clarify

Reply:
Wasn't sure of the exact wording, i followed the existing format. I've added a little bit but not sure if it's enough. If insufficient, it would help to have some more guidance.